lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php

```diff
@@ -624,7 +624,7 @@ private function completeIdGeneratorMapping(ClassMetadataInfo $class): void
 
                 $sequenceGenerator = new SequenceGenerator(
                     $this->em->getConfiguration()->getQuoteStrategy()->getSequenceName($definition, $class, $this->getTargetPlatform()),
-                    $definition['allocationSize']
+                    (int) $definition['allocationSize']
```

Comment:
I'd revert this change as well. Let's not change behavior unless we really hit a bug.

Reply:
Yes, let's baseline the errors and do the behavior changes on 2.10.

Reply:
Done.